tmt/steps/discover/fmf.py

```diff
@@ -43,26 +43,44 @@ class DiscoverFmfStepData(tmt.steps.discover.DiscoverStepData):
         default=cast(Optional[str], None),
         option=('-u', '--url'),
         metavar='REPOSITORY',
-        help='URL of the git repository with fmf metadata.')
+        help="""
+            Git repository containing the metadata tree.
+            Current git repository used by default.
+            """)
     ref: Optional[str] = field(
         default=cast(Optional[str], None),
         option=('-r', '--ref'),
         metavar='REVISION',
-        help='Branch, tag or commit specifying the git revision.',
+        help="""
+            Branch, tag or commit specifying the desired git
+            revision. Defaults to the remote repository's default
+            branch if url given or to the current ``HEAD`` if url
+            not provided.
+            """,
         normalize=normalize_ref)
     path: Optional[str] = field(
         default=cast(Optional[str], None),
         option=('-p', '--path'),
         metavar='ROOT',
-        help='Path to the metadata tree root.')
+        help="""
+            Path to the metadata tree root. Must be relative to
+            the git repository root if url provided, absolute
```

Comment:
"url" should be marked as a literal, referencing to ``url`` key.

```
the git repository root if ``url` was provided, absolute
```

Reply:
Updated !